litellm/integrations/athina.py

```diff
@@ -78,10 +79,12 @@ def log_event(self, kwargs, response_obj, start_time, end_time, print_verbose):
             # Add additional metadata keys
             metadata = kwargs.get("litellm_params", {}).get("metadata", {})
             if metadata:
+                print("additional_keys", self.additional_keys)
                 for key in self.additional_keys:
+                    print("key", key)
                     if key in metadata:
+                        print("key is being added", key)
```

Comment:
there are prints here, did you review your own code before asking us @vivek-athina ? 

- please add unit tests in tests/litellm/integrations/test_athina.py